atomic_reactor/plugins/post_import_image.py

```diff
@@ -29,7 +29,7 @@ class ImportImagePlugin(PostBuildPlugin):
 
     def __init__(self, tasker, workflow, imagestream, docker_image_repo,
                  url, build_json_dir, verify_ssl=True, use_auth=True,
-                 insecure_registry=None, retry_delay=30, import_attempts=3):
+                 insecure_registry=None):
```

Comment:
Let's remove reference to these params in docstring as well.

Reply:
done